services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/MetersSpec.groovy

```diff
@@ -358,6 +359,28 @@ class MetersSpec extends BaseSpecification {
         ]
     }
 
+    @Ignore("https://github.com/telstra/open-kilda/issues/2043")
+    def "Try to sync meters for unmetered flow"() {
+        given: "A flow with custom meter rate and burst, that differ from defaults"
+        def randomSwitches = topology.activeSwitches
+        def src = randomSwitches[0]
+        def dst = randomSwitches[1]
+        assumeTrue("Desired switch combination is not available in current topology", src && dst)
+
+        and: "A flow with the 'bandwidth: 0' and 'ignoreBandwidth: true' fields, that differ from defaults"
+        def flow = flowHelper.randomFlow(src, dst)
+        flow.ignoreBandwidth = true
+        flow.maximumBandwidth = 0
+        flowHelper.addFlow(flow)
+
+        when: "Update burst parameter in meter"
```

Comment:
This may be misunderstood. We are not actually updating the burst, but rather `resetting meter burst and rate to default` or `syncing the meter with defaults` etc.

Reply:
You are right
I'll change it